src/Fantomas.Core/CodePrinter.fs

```diff
@@ -3234,9 +3234,12 @@ let genTypeDefn (td: TypeDefn) =
                 +> genSingleTextNode node.ClosingBrace
 
             let genMembers =
-                onlyIf hasMembers sepNln
-                +> sepNlnBetweenTypeAndMembers typeDefnNode
-                +> genMemberDefnList members
+                onlyIf
+                    hasMembers
+                    (sepNln
+                     +> sepNlnBetweenTypeAndMembers typeDefnNode
+                     +> genMemberDefnList members
+                     +> onlyIfCtx (fun ctx -> ctx.Config.IsStroustrupStyle) unindent)
```

Comment:
A bit of a nit but would this not make more sense to have it closer to the `indent`?
Something like:
```
                genAccessOpt node.Accessibility
                +> genRecordFields
                +> onlyIf hasMembers (sepSpace +> withKw +> indent +> genMembers +> unindent)
```

Reply:
Good call.